concordium-node/build.rs

```diff
@@ -3,9 +3,9 @@ use std::{env, path::Path};
 use std::{process::Command, str};
 
 #[cfg(all(not(feature = "static"), target_os = "linux"))]
-const GHC_VARIANT: &str = "x86_64-linux-ghc-8.10.4";
+const GHC_VARIANT: &str = "x86_64-linux-ghc-9.0.2";
 #[cfg(all(not(feature = "static"), target_os = "macos"))]
-const GHC_VARIANT: &str = "x86_64-osx-ghc-8.10.4";
+const GHC_VARIANT: &str = "aarch64-osx-ghc-9.0.2";
```

Comment:
This change means that the node will not properly build on an Intel mac.

We need another solution with more targeted platforms.

Reply:
Good catch! 👍